net.nokyan.Resources.json

```diff
@@ -0,0 +1,139 @@
+{
+    "id": "net.nokyan.Resources",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "45",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-nightly",
+        "org.freedesktop.Sdk.Extension.llvm16"
+    ],
+    "command": "resources",
+    "finish-args": [
+        "--device=dri",
+        "--env=G_MESSAGES_DEBUG=none",
+        "--env=RUST_BACKTRACE=full",
+        "--env=RUST_LOG=resources=debug",
+        "--env=XDG_DATA_DIRS=/app/share:/usr/share:/usr/share/runtime/share:/run/host/user-share:/run/host/usr/share:/run/host/share:/var/lib/flatpak/exports/share:~/.local/share/flatpak/exports/share",
+        "--filesystem=/var/lib/flatpak/app:ro",
+        "--filesystem=/var/lib/flatpak/exports/share:ro",
+        "--filesystem=xdg-data/flatpak/app:ro",
+        "--filesystem=xdg-data/flatpak/exports/share:ro",
+        "--filesystem=host",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.Flatpak"
```

Comment:
this permission is not allowed by default.

Reply:
I have already opened a pull request for that https://github.com/flathub/flatpak-builder-lint/pull/189